client/extensions/woocommerce/app/settings/shipping/shipping-zone/shipping-methods/flat-rate.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import FormCheckbox from 'components/forms/form-checkbox';
+import FormCurrencyInput from 'components/forms/form-currency-input';
+import FormFieldSet from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import {
+	setShippingIsTaxable,
+	setShippingCost
+} from 'woocommerce/state/ui/shipping/zones/methods/flat-rate/actions';
+
+const FreeShippingMethod = ( { id, siteId, cost, tax_status, translate, actions } ) => {
+	const isTaxable = 'taxable' === tax_status;
+	const onTaxableChange = () => ( actions.setShippingIsTaxable( siteId, id, ! isTaxable ) );
+	const onCostChange = ( event ) => ( actions.setShippingCost( siteId, id, event.target.value ) );
+
+	const renderCostInput = () => {
+		//TODO: remove hardcoded currency settings
+		return <FormCurrencyInput
+			currencySymbolPrefix={ '$' }
+			currencySymbolSuffix={ '' }
+			value={ cost }
+			onChange={ onCostChange } />;
+	};
+
+	return (
+		<div>
+			<FormFieldSet>
+				<FormLabel>{ translate( 'Cost:' ) }</FormLabel>
```

Comment:
Hi! I've found a possible matching string that has already been translated **59** times:
`translate( 'Cost' )` *ES Score: 14*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/14275-shipping-methods-ui)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).